lesson_06/homework_06.py

```diff
@@ -35,6 +63,15 @@
 Введіть число: 12345
 Сума цифр числа 12345: 15
 """
+users_number = input('Print your number here: ')
+user_number_list = list(users_number)
+total_count = 0
```

Comment:
```suggestion
user_number_list = [int(i) for i in users_number]
total_count = sum(users_number)
```
і все, далі нічого не треба